torch/distributed/optim/utils.py

```diff
@@ -24,6 +24,18 @@
     optim.Adamax: _FunctionalAdamax,
 }
 
+
```

Comment:
In case you meant to address this but missed it:
Could we move this into a triple quote docstring under `def register_functional_optim(key, optim):`?